src/main/java/com/example/playdocker/services/MessageService.java

```diff
@@ -0,0 +1,198 @@
+package com.example.playdocker.services;
+
+import com.example.playdocker.Utils.MessageUtils;
+import com.example.playdocker.models.Message;
+import com.example.playdocker.models.Score;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class MessageService {
```

Comment:
Rozumiem, że wydało Ci się prostsze użycie plików CSV do symulowania zapisywania/odczytywania danych (i ich persystencji) ale uwierz mi, że to nie jest prostsze rozwiązanie :) Gdybyś teraz chciał podłączyć bazę to musiałbyś napisać wszystko od nowa. 

Korzystasz ze Springa, wykorzystaj to :) mógłbyś dodać jedną linijką repository, które umożliwi Ci zapis twoich modeli do bazy, bez żadnych dodatkowych appendów do plików. Sama baza nie musi być również stawiana jako osobna instancja, mozesz użyć H2/HSQLDB i wielu innych in-memory db. Naprawdę warto poczytać dobrą książkę o Springu i projektowaniu aplikacji - dużo się nauczysz i zrozumiesz dlaczego to podejście nie zda egzaminu.